headers/fmi3Functions.h

```diff
@@ -149,13 +165,13 @@ Functions for FMI3 for Model Exchange
 Functions for FMI3 for Co-Simulation
 ****************************************************/
 
-#define fmi3SetRealInputDerivatives      fmi3FullName(fmi3SetRealInputDerivatives)
-#define fmi3GetRealOutputDerivatives     fmi3FullName(fmi3GetRealOutputDerivatives)
+#define fmi3SetInputDerivatives          fmi3FullName(fmi3SetInputDerivatives)
+#define fmi3GetOutputDerivatives         fmi3FullName(fmi3GetOutputDerivatives)
 #define fmi3DoStep                       fmi3FullName(fmi3DoStep)
 #define fmi3CancelStep                   fmi3FullName(fmi3CancelStep)
 #define fmi3GetStatus                    fmi3FullName(fmi3GetStatus)
-#define fmi3GetRealStatus                fmi3FullName(fmi3GetRealStatus)
-#define fmi3GetIntegerStatus             fmi3FullName(fmi3GetIntegerStatus)
+#define fmi3GetDoubleStatus              fmi3FullName(fmi3GetDoubleStatus)
```

Comment:
Dito for fmi3GetFloat64Status

```suggestion
#define fmi3GetFloat64Status              fmi3FullName(fmi3GetFloat64Status)
```